tests/tests/test-issuers.spec.js

```diff
@@ -0,0 +1,146 @@
+const ISSUER_NAME = `test-issuer-${Math.floor(Math.random() * 9999) + 1000}`;
+const SECRET_NAME = `issuer-secret-${Math.floor(Math.random() * 9999) + 1000}`;
+
+context('Test Issuers', () => {
+  before(() => {
+    cy.loginAndSelectCluster();
+    cy.goToNamespaceDetails();
+
+    cy.getLeftNav()
+      .contains('Configuration')
+      .click();
+
+    cy.getLeftNav()
+      .contains('Secrets')
+      .click();
+
+    cy.getIframeBody()
+      .contains('Create Secret')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Secret Name"]:visible')
+      .type(SECRET_NAME);
+
+    cy.getIframeBody()
+      .contains('.fd-dialog__content button', 'Create')
```

Comment:
It should be 
`    .find('[role=dialog]')
      .contains('button', 'Create')`